tests/foreman/ui_airgun/test_host.py

```diff
@@ -510,3 +524,764 @@ def test_positive_export(session):
                      row['Environment'])
                 )
         assert set(actual_fields) == expected_fields
+
+
+@tier3
+def test_positive_create_with_inherited_params(session):
+    """Create a new Host in organization and location with parameters
+
+    :BZ: 1287223
+
+    :id: 628122f2-bda9-4aa1-8833-55debbd99072
+
+    :expectedresults: Host has inherited parameters from organization and
+        location
+
+    :CaseImportance: High
+    """
+    org = entities.Organization().create()
+    loc = entities.Location(organization=[org]).create()
+    org_param = dict(name=gen_string('alphanumeric'), value=gen_string('alphanumeric'))
+    loc_param = dict(name=gen_string('alphanumeric'), value=gen_string('alphanumeric'))
+    host_template = entities.Host(organization=org, location=loc)
+    host_template.create_missing()
+    host_name = u'{0}.{1}'.format(host_template.name, host_template.domain.name)
+    with session:
+        session.organization.update(org.name, {'parameters.resources': org_param})
+        session.location.update(loc.name, {'parameters.resources': loc_param})
+        session.organization.select(org_name=org.name)
+        session.location.select(loc_name=loc.name)
+        create_fake_host(session, host_template)
+        values = session.host.read(host_name, 'parameters')
+        expected_params = {(org_param['name'], org_param['value']),
+                           (loc_param['name'], loc_param['value'])}
+        assert expected_params.issubset(
+            {(param['name'], param['value']) for param in values['parameters']['global_params']}
+        )
+
+
+@tier3
+def test_negative_delete_primary_interface(session, module_host_template):
+    """Attempt to delete primary interface of a host
+
+    :id: bc747e2c-38d9-4920-b4ae-6010851f704e
+
+    :BZ: 1417119
+
+    :expectedresults: Interface was not deleted
+
+    :CaseLevel: System
+    """
+    name = gen_string('alpha').lower()
+    host_name = '{0}.{1}'.format(name, module_host_template.domain.name)
+    interface_id = gen_string('alpha')
+    with session:
+        create_fake_host(session, module_host_template, interface_id=interface_id,
+                         extra_values={'host.name': name})
+        session.host.delete_interface(host_name, interface_id)
```

Comment:
did you get error on UI? any pop-up? such validation would be much better

Reply:
the delete button is disabled , and we click on that button, and nothing happed
```
019-02-26 11:05:34 - widgetastic_null - DEBUG - click: Text(".//button[contains(., 'Delete')]")
2019-02-26 11:05:34 - widgetastic_null - DEBUG - move_to_element: Text(".//button[contains(., 'Delete')]")
2019-02-26 11:05:34 - widgetastic_null - WARNING - [HostEditView/interfaces/interfaces_list[0]]: __locator__ of TableRow class returns a WebElement!
2019-02-26 11:05:34 - widgetastic_null - WARNING - [HostEditView/interfaces/interfaces_list[0]['Actions']]: __locator__ of TableColumn class returns a WebElement!
2019-02-26 11:05:35 - widgetastic_null - INFO - [HostEditView/interfaces/interfaces_list[0]['Actions']/TableActions/delete]: click (elapsed 177 ms)
2019-02-26 11:05:35 - widgetastic_null - DEBUG - [HostEditView/submit]: click started
```

Reply:
then it is hard to understand whether we have issue with application or just problem with selenium that cannot click button due any reason

Reply:
resolved in last commit and in dependency https://github.com/SatelliteQE/airgun/pull/295